sdk/storage/azure-storage-blob/src/main/java/com/azure/android/storage/blob/StorageBlobClient.java

```diff
@@ -508,6 +512,73 @@ BlobDeleteResponse deleteWithRestResponse(String containerName,
             cancellationToken);
     }
 
+    /**
+     * Gets tags associated with a blob.
+
+     * @param containerName     The container name.
+     * @param blobName          The blob name.
+     * @return The blob's tags.
+     */
+    public Map<String, String> getBlobTags(String containerName,
+                                           String blobName) {
+        BlobTags response = this.storageBlobServiceClient.getTags(containerName,
+            blobName);
+        Map<String, String> tags = null;
+        if (response.getBlobTagSet() != null) {
```

Comment:
Is there any way we can push this conversion into the intern method?

Reply:
I hesitate to do that cause if/when there is a generator for Android it will probably replace everything in the Impl file, so I want to keep it as "generated" as possible

Reply:
Discussed offline, going to move this into a ModelHelper class